.evergreen/config/test-variants.yml

```diff
@@ -0,0 +1,116 @@
+#
+# Test variants - only run for select platforms worth testing on
+#
+buildvariants:
+  # PHP 8.2: test all topologies on all versions of MongoDB
+  - name: test-debian11-php82-local
+    tags: ["test", "debian", "x64"]
+    display_name: "Test: Debian 11, PHP 8.2"
+    run_on: debian11-small
+    expansions:
+      FETCH_BUILD_VARIANT: "build-debian11"
+      FETCH_BUILD_TASK: "build-php-8.2"
+      PHP_VERSION: "8.2"
+      VARIANT: debian11 # Referenced by ADL build script for downloading MQLRun
+    depends_on:
+      - variant: "build-debian11"
+        name: "build-php-8.2"
+    tasks:
+      - ".standalone .local !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
+      - ".replicaset .local !.csfle !.3.6 !.4.0 !.4.2 !.4.4 !.5.0"
```

Comment:
Should all of these lines include `!.csfle`? As-is, this assumes that `templates/test/csfle.yml` will only apply to replica set topologies, which could change in the future.

Reply:
Excluded this. As unlikely as changes are, this prevents surprises down the line.